Modules/System/Base64 Convert/README.md

```diff
@@ -107,6 +184,29 @@ The string to convert.
 #### Return Value
 *[Text](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/text/text-data-type)*
 
+Regular string that is equivalent to the input base-64 string.
+### FromBase64 (Method) <a name="FromBase64"></a> 
+The length of Base64String, ignoring white-space characters, is not zero or a multiple of 4.
```

Comment:
...multiple of four.

Reply:
That's just a copy'paste of Microsoft's text from the existing methods..